netlogo-gui/src/test/headless/TestLanguage.scala

```diff
@@ -180,3 +191,8 @@ TurtleSet_2D
     assert(tests.toString == "List(LanguageTest(test,TurtleSet_2D,List(Command(crt 1,Observer,Success()), Reporter([turtle-set self] of turtle 0 = turtles,Success(true)))))")
   }
 }
+
+class LanguageTestException (line: String, cause: Exception)
+extends Exception("The test errored on the following line: " + line, cause) {
+  setStackTrace(Array())
```

Comment:
Does this suppress the original exception stack trace in the output? Sometimes that contains important information.